### ISSUE TITLE
Add Code Formatter to follow defined Code Style

### ISSUE THREAD

**Issue by Andre0711er** [Cypher01/ta4j-extended#2]
Hi.

Can you add your used Code Formatting Rules. I personally prefer it for IntelliJ IDEA, but whatever you have, it will help.
So ta4j project use Eclipse Code-Formatter XML. Might be, you can have a look there and use this.

Thanks,

André.

**Comment by Cypher01** [comment]
I also use IntelliJ IDEA for auto-formatting. To be honest, it's been ages since I configured it, but what I definitely changed is to use tabs as the default indent for all languages.

Until recently, I didn't know the `formatter-maven-plugin` used by the ta4j project, it seems to be a pretty useful tool to make code formatting independend of any IDE. Maybe I will add this Maven plugin to ta4j-extended, but I don't know when I have time for it.

**Comment by Cypher01** [comment]
I hope I didn't sound too harsh on your last PR, it's just not nice when the Git history shows almost all lines changed, simply to an automatic conversion from tabs to spaces. ;-)

**Comment by Andre0711er** [comment]
No, it's understandable that you take care of your code style. i will do the same with my code and i have my own code style. ;-)
Can you export your intellij format settings and add to your project? I will reformat my code with that code style.

Thanks.

**Comment by Andre0711er** [comment]
As Information: In my Fork (https://github.com/Andre0711er/ta4j-extended) i just added the code formatter with plugin which is used on ta4j parent project
is deadly easy to use it:

mvn formatter:format

cleans whole code based on defined formatter xml file.
might be a good idea, to bring it on same level as ta4j main project.

**Comment by Cypher01** [comment]
I'll take a look soon :+1:

**Comment by Cypher01** [comment]
This was integrated by PR #8
